src/packages.jl

```diff
@@ -239,7 +237,7 @@ For details, please refer to its documentation.
 function install(spec::String, version::String = "";
                                interactive::Bool = true, quiet::Bool = false,
                                debug::Bool = false,
-                               pkgdir::AbstractString = DEFAULT_PKGDIR[])
+                               pkgdir::AbstractString = GAP.sysinfo["DEFAULT_PKGDIR"])
```

Comment:
```suggestion
                               pkgdir::AbstractString = sysinfo["DEFAULT_PKGDIR"])
```
this is a submodule of `GAP`, but `sysinfo` gets already imported

Reply:
Sure, but I'd like to keep this in sync with the docstring, to avoid confusion